lib/functions/plugins.bash

```diff
@@ -79,8 +79,8 @@ plugin_add_command() {
   mkdir -p "$(asdf_data_dir)/plugins"
 
   if [ -d "$plugin_path" ]; then
-    display_error "Plugin named $plugin_name already added"
-    exit 2
+    printf '%s\n' "Plugin named $plugin_name already added" >&2
```

Comment:
looks like it should be either without passing to stderr (better imho) or without changes

Reply:
Agreed, it would be better to print to standard output here